python/hail/linalg/matrix.py

```diff
@@ -122,6 +162,42 @@ def __sub__(self, that):
     def dot(self, that):
         return BlockMatrix(self._jbm.multiply(that._jbm))
 
+    def entries_table(self):
+        """Returns a coordinate Table with a row for each entry in the BlockMatrix.
```

Comment:
Suggestion: "Returns a table with the coordinates and numeric value of each block matrix entry."